test/util/resourceValidationUtils.test.js

```diff
@@ -0,0 +1,59 @@
+const { retrieveProfiles, getValidationInfo } = require('../../src/util/resourceValidationUtils');
+
+describe('resourceValidationInfo tests', () => {
+  test('retrieveProfiles returns Calculation on dollar-sign operation', () => {
+    const profile = retrieveProfiles('/4_0_1/Measure/$dataRequirements', {});
+    expect(profile).toEqual('Calculation');
+  });
+  test('retrieveProfiles returns Bundle on transaction bundle upload URL', () => {
+    const profile = retrieveProfiles('/4_0_1', {});
+    expect(profile).toEqual(['Bundle']);
+  });
+  test('retrieveProfiles returns Patient with Patient URL', () => {
+    const profile = retrieveProfiles('/4_0_1/Patient', {});
+    expect(profile).toEqual(['Patient'], {});
+  });
+  test('retrieveProfiles returns Patient and meta.profile info with Patient URL and meta.profile in body', () => {
+    const profile = retrieveProfiles('/4_0_1/Patient', { meta: { profile: ['testProfile', 'testProfile2'] } });
+    expect(profile).toEqual(['testProfile', 'testProfile2', 'Patient'], {});
+  });
+  test('getValidationInfo returns correct object on valid FHIR response', async () => {
+    const axios = require('axios');
```

Comment:
Do we need to require axios twice (here and below) in order for the mocks to works expected? If so I think there's a better solution we can do where we only require axios once at the top of the file, and then do an `afterEach` hook to clear all configured mocks in between tests